rmgpy/data/kinetics/common.py

```diff
@@ -90,10 +90,6 @@ def sortEfficiencies(efficiencies0):
         #Write out additional data if depository or library
         #kinetic rules would have a Group object for its reactants instead of Species
         if isinstance(entry.item.reactants[0], Species):
-            if entry.label != str(entry.item):
```

Comment:
@goldmanm does the fact that I'm deactivating this failsafe bother you?  I believe it's suppose to prevent rmg from outputting a seed with non-unique names, but it does so by requiring that they be the string names, which I would argue is too restrictive.